index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="eng" dir="ltr">
+
+<head>
+	<meta charset="utf-8">
+	<title>Apple</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+<main>
+
+<div class="container">			
+
+	<nav class="globalheader">
+           <ul class="globalnav">
+            <li class="gn-apple"><a href="#"><span>Apple</span></a></li>
+            <li class="gn-store"><a href="#"><span>Store</span></a></li>
+            <li class="gn-mac"><a href="#"><span>Mac</span></a></li>
+            <li class="gn-ipod"><a href="#"><span>iPod</span></a></li>
+            <li class="gn-iphone"><a href="#"><span>iPhone</span></a></li>
+            <li class="gn-ipad"><a href="#"><span>iPad</span></a></li>
+            <li class="gn-itunes"><a href="#"><span>iTunes</span></a></li>
+            <li class="gn-support"><a href="#"><span>Support</span></a></li>
+          </ul>
+          <div class="globalsearch">
+            <form class="g-search">
+                <div class="magnify">
+                    <img src="images/search-icon.png" alt="search" height="13" width="13">
+                </div>
+            </form>
```

Comment:
Kindly change this into a search input field. Also fix the HTML validation errors in the HTML code with this [tool](https://validator.w3.org/nu/#file).